source/lac/trilinos_sparse_matrix.cc

```diff
@@ -2764,6 +2764,76 @@ namespace TrilinosWrappers
 
 
       TrilinosPayload
+      TrilinosPayload::identity_payload () const
+      {
+        TrilinosPayload return_op (*this);
+
+        return_op.vmult = [](Range &tril_dst,
+                             const Range &tril_src)
+        {
+          tril_dst = tril_src;
+        };
+
+        return_op.Tvmult = [](Range &tril_dst,
+                              const Range &tril_src)
+        {
+          tril_dst = tril_src;
+        };
+
+        return_op.inv_vmult = [](Range &tril_dst,
+                                 const Range &tril_src)
+        {
+          tril_dst = tril_src;
+        };
+
+        return_op.inv_Tvmult = [](Range &tril_dst,
+                                  const Range &tril_src)
+        {
+          tril_dst = tril_src;
+        };
+
+        return return_op;
+      }
+
+
+
+      TrilinosPayload
+      TrilinosPayload::null_payload () const
+      {
+        TrilinosPayload return_op (*this);
+
+        return_op.vmult = [](Range &tril_dst,
+                             const Domain &)
+        {
+          tril_dst.Scale(0);
```

Comment:
No, unfortunately `Epetra_MultiVector` does not have a scalar overload for `operator=`. But I see that we [do things slightly differently](https://github.com/dealii/dealii/blob/master/include/deal.II/lac/trilinos_vector_base.h#L1228) for our `operator=(scalar)`, so I'll adjust this accordingly.